### DISCUSSION TITLE
Trouble rendering html

### DISCUSSION THREAD

**Discussion by eriskar** [quarto-dev/quarto-cli#4895] [Q&A]
Hey, guys

Sorry if this has been answered elsewhere (I looked but couldn't find the solution), but I am struggling rendering my .qmd file featuring python code to html file.

This is my yaml:

---
title: "generic title"
author: generic author name
date: March 18, 2023
toc: true
format:
  html:
    embed-resources: true
execute:
  eval: false
  echo: true
---

It only produces a .ipynb file for some reason while Quarto provides me with this error message (the filepaths in the error message have been altered by me):

PS C:\[file path]\Skrivebord> quarto preview "c:/[file-path-to-quarto-file].qmd" --to html --no-browser --no-watch-inputs
ERROR: Ingen tilgang til filen. (os error 1920), stat 'C:\[file-path-to-python]\python3.10.exe'
PS C:\[file-path-to-desktop]\Skrivebord>

**Comment by rgaiacs** [comment]
Hi @eriskar, `os error 1920` is because quarto can't access your Python installation. Sounds that Windows is blocking the access or the Python installation doesn't exist.

**Reply by eriskar** [reply]
Thank you so much for the reply -- greatly appreciated! That's strange. I don't want to run the Python code, just show it as a part of the narrative text in the html file (cf. execute: eval: false echo: true). This happened over night and I've used the yaml code a million times, so I have no idea what changed.

**Reply by cscheid** [reply]
can you run `quarto check`?

I think what's happening here is that, even if every quarto cell is `execute: eval: false echo: true`, quarto still attempts to detect and start the engine if you have

````
```{python}
...
```
````

in your code.

If you only want to display code, you could instead

````
```python
...
```
````

Note the difference in brackets.

With that said, I don't think this behavior hasn't changed in recent versions of quarto, so I don't know what accounts for your recent change. This is why we always ask for "minimal, fully reproducible examples", and not fragments of them. We need to be able to see the whole file (and whether it's in a book or website project, etc)  to understand the context.

**Reply by eriskar** [reply]
Hi,

First of all, sorry for not providing reproducable results. I completely forgot and won't happen again. Sorry!

Second, it seems that removing {} solved the issue, so that suffices to make me happy. Don't know what has happened that made the change occur, but as long as Quarto works, I'm a happy man.

Thank you so much for the help!
